examples/README.md

```diff
@@ -0,0 +1,33 @@
+# `recOrder` example scripts
+
+These scripts demonstrate three types of reconstruction:
+
+1. 3D brightfield data to 3D phase in `3D-bf-to-3D-phase.py`. 
+2. 3D brightfield data to 2D phase in `3D-bf-to-2D-phase.py`.
+3. 4D polarization data to retardance, orientation, and 3D phase in `recon-qlipp.py`.   
+
+## Getting started
+First, [install `recOrder`](../docs/software-installation-guide.md) and run these scripts with 
+```bash
+python <script-name.py>
+```
+Running any of these scripts without modification will test your installation by running a reconstruction on random data. A successful run will open a `napari` window with results reconstructed from random data. 
+
+Next, [download the test data from zenodo (47 MB)](https://zenodo.org/record/6983916/files/recOrder_test_data.zip?download=1), and modify the script to load the test data. For example, in `3D-bf-to-2D-phase.py`, replace `/path/to/ome-tiffs/or/zarr/store/` with `/path/to/recOrder_test_data/2022_08_04_recOrder_pytest_20x_04NA_BF/2T_3P_16Z_128Y_256X_Kazansky_BF_1`.
+
+Finally, modify the script to process your data. Start by loading your data (our readers currently support `.tiff`, `ome.tiff`, `NDTiff`, and `.zarr`, but any `numpy` array will work). Fill in your imaging parameters (or connect your metadata to these parameters), and prototype with a script. 
+
+We recommend prototyping a reconstruction with a single position and time point. You may need to test several regularization parameters to find a value that yields results that aren't too noisy or too smooth. 
+
+After prototyping, the script can be applied to multiple datasets with a python `for` loop (slowest), `multiprocessing` (faster), or `slurm` (fastest). 
```

Comment:
`slurm` is not always available to all the users. We might want to reword it as `batch processing with HPC job schedulers such as Slurm (fastest)`

Reply:
Nice suggestion. Thanks!